testing/trino-benchto-benchmarks/src/main/resources/sql/presto/tpcds/q91.sql

```diff
@@ -23,7 +23,7 @@ WHERE ("cr_call_center_sk" = "cc_call_center_sk")
          AND ("cd_education_status" = 'Unknown'))
       OR (("cd_marital_status" = 'W')
          AND ("cd_education_status" = 'Advanced Degree')))
-   AND ("hd_buy_potential" LIKE 'Unknown')
+   AND ("hd_buy_potential" LIKE 'Unknown        ')
```

Comment:
This should be `Unknown%`

Reply:
@sopel39 since you know why, can you please make this change?

Reply:
It's already fixed

Reply:
https://github.com/trinodb/trino/pull/8430